### PR TITLE
Point 14 in todays list - new bullet point in gateway considerations

### PR DESCRIPTION
**Pull request by gunnarhm**

### CODE REVIEW

**Review by cdh4u — COMMENTED** [review]
Looks good, but you need to use <xref target="T140ad1"/> for the reference.

**Comment by gunnarhm** [comment]
I do not understand the comment.  Is a word missing? 
Have you done the extra correction you seem to indicate is needed?

**Comment by cdh4u** [comment]
> I do not understand the comment.  Is a word missing?
> Have you done the extra correction you seem to indicate is needed?

You need to use the proper XML code for the T140ad1 reference.

File: draft-ietf-mmusic-t140-usage-data-channel.xml [hunk 1]
```diff
@@ -566,6 +566,12 @@ Answer:
             the gateway SHOULD insert the T.140 missing text marker <xref target="T140ad1"/> in the data sent on the outgoing RTP stream
             if it detects or suspects that data on the T.140 data channel was lost.
           </t>
+          <t>
+            If the gateway detects that the T.140 data channel has failed and got torn down, once the data channel
+            has been reestablished the gateway SHOULD insert the T.140 missing text marker [T140ad1] in the data
```
Comment (gunnarhm) [inline draft-ietf-mmusic-t140-usage-data-channel.xml]: "[T140ad1]"  s.b. replaced with  ".lt.xref target="T140ad1"/.gt." Wher .gt. stands for > and .lt. stands for <

Comment (cdh4u) [reply draft-ietf-mmusic-t140-usage-data-channel.xml]: Looks good.

**Pull request merged**